src/middleware/authentication.middleware.ts

```diff
@@ -22,7 +22,7 @@ export const authentication = (req: Request, res: Response, next: NextFunction):
       let returnUrl = SOLD_LAND_FILTER_URL;
 
       if (req.path.startsWith(UPDATE_LANDING_URL)) {
-        returnUrl = SECURE_UPDATE_FILTER_URL;
+        returnUrl = UPDATE_CONTINUE_WITH_SAVED_FILING_URL;
```

Comment:
This only if save&resume is enabled. The page is only to be shown if the feature is on.